pkg/vetter/util/util.go

```diff
@@ -211,7 +172,7 @@ func GetMeshConfig(cm *corev1.ConfigMap) (*meshv1alpha1.MeshConfig, error) {
 		return nil, nil
 	}
 	var cfg meshv1alpha1.MeshConfig
-	if err := ApplyYAML(c, &cfg); err != nil {
+	if err := ApplyYAML(c, &cfg, false); err != nil {
```

Comment:
Double check that we need both `ApplyYAML` and the other YAML functions in the inject utils